code/src/Artist.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import './artist.css'
+
+export const Artist = (props) => {
+    console.log("artist props", props)
+        return (
+            <a className="artist-name" href={props.href}>{props.name}</a>
+        )

```

Comment:
Keep an eye on your indentation! These 3 lines should be one level less indented